Skills/Perk/PregnancyMotherOfTheYear.gd

```diff
@@ -0,0 +1,24 @@
+extends PerkBase
+
+func _init():
+	id = Perk.PregnancyMotherOfTheYear
+	skillGroup = Skill.Pregnancy
+
+func getVisibleName():
+	return "Mother of the year"
+
+func getVisibleDescription():
+	return "AlphaCorp can't thank you enough for providing more cannon fodder for their global operations. Oh! we meant to say what a nice kids you have we always happy to help them grow and prosper, right! \n" \
+			+ "You get a stats boost buff for 2 days after giving birth. \n" \
+			+ "You get a credits payment based on the amout of children you provide."
```

Comment:
5 credits per birth + 2 per child up to a total limit of 30-40 credits per birth or so.

Reply:
5 is too much x3. Basically the way I see it, 1 work credit = 1 work day. And breeding is not that hard, make it 1 credit or something x3.

I'm not sure if giving credits is a good idea for a perk. The stat boost would be enough

Reply:
1 work day = 1 credit is interesting, why does empty plastic bottle cost 5 credits then? xD Also prostitutes charge like 5-8 credits up to 20+ in some cases. So there should be either a fractions of a credit used and prices lowered significantly for the most items or increase it to something like 5 credits per workday.

Reply:
Default preg time is 5 days, so 1 is kinda low for tier 3 perk which you can get only by giving birth plenty of times. Maybe 3 + 1 per child with a limit of 20?

Reply:
Everyone is greedy and AlphaCorp really doesn't want to give inmates credits ^^. Thats why inmates/vendomats can ask for a lot while the corporation gives very little. I dunno, I would still just keep the stats boost and remove the credits reward

You know what would be cool? A 'bounty' board in the nursery that generates random tasks like "Give birth/be a father to 3 cats" or "give birth to 5 males" or whatever. Tasks that ask for hybrids could pay more. And basically, even if you're not the right species, you can still make that happen with strapons and cum transfering x3. Could also have tasks like "donate cum/breastmilk"

So basically I think giving credits for kids is too easy, needs more challenge.. But I like the temporary stats boost ^^

Reply:
That nursery board is a nice idea indeed. Although it should be unrelated to perks IMHO. 
There is a difference still between greed and a comical level of greed where an empty bottle costs 5 credits but you get paid 1 for a work day. Also I had the impression that AlphaCorp is really interested in inmates producing more children for their war efforts or whatever, that's why I was wondering why condoms and infertility pills are legal. Maybe make some shady character to sell all this stuff like condoms, anaphrodisiac pills, birth control pills and if you are caught by the guards with them you get fed breeder pill and put into stocks.

Reply:
Lowered to 2 credits per birth + 1 per child (limit is 20) for now.